site/content/docs/11-deploying.md

```diff
@@ -61,4 +61,70 @@ When deploying to [Now][], you can pass the environment variable into Now itself
 now -e SAPPER_TIMESTAMP=$(date +%s%3N)
 ```
 
-[Now]: https://zeit.co/now
\ No newline at end of file
+[Now]: https://zeit.co/now
+
+
+## Deploying with Docker
+
+[Docker](https://docs.docker.com/install/) can be used to build an image and run it on your local system. Doing this locally allows testing the build process before building and deploying it on a cloud platform. The Docker image can be build by adding a `Dockerfile` into your project:
+
+```Dockerfile
+# Dockerfile
+FROM mhart/alpine-node:latest as build
+
+WORKDIR /app
+COPY . .
+RUN npm ci
+RUN npm run build
+
+FROM mhart/alpine-node:latest as prod
+
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm ci --production
+
+FROM mhart/alpine-node:slim
```

Comment:
I'm not really sure we need to use the slim image here, I've been successfully using alpine images for the release container too. If anything it is more important that your release container is as small as possible, since that is the one that will be passed around and replicated the most.